src/audio/audio.h

```diff
@@ -121,67 +125,74 @@ class Audio
  *
  * @see Audio
  */
-class NoAudio : public Audio
+class Null_audio : public Audio
```

Comment:
don't like this style of naming of types. What was wrong with `NullAudio` or even `Null_Audio` ? consistent capitalisation, imo

Reply:
Mnn, I'm half inclined to agree.  I was trying to follow [core guidelines NL.10](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-camel) with its suggestion of Stroustrup-style 'underscore but capitalise first letter', but I do agree that it looks ugly, and one could argue that playd already had a convention with reasoning.

Tbh, with half-decent namespacing, there won't actually be many instances in playd where things need long and complicated class names, so…

Reply:
The vast majority of that rule is very much "don't do this if you already have an existing style, so I still don't agree with changing everything to match

Reply:
meh, I figured that the codebase was small enough to be able to push through the whole rename in one go, but meh, I can revert